modules/images/webp-uploads/load.php

```diff
@@ -35,5 +35,186 @@ function webp_uploads_filter_image_editor_output_format( $output_format, $filena
 
 	return $output_format;
 }
-add_filter( 'image_editor_output_format', 'webp_uploads_filter_image_editor_output_format', 10, 3 );
```

Comment:
Nit-pick, but let's keep the `add_filter` / `add_action` calls right below the callback function since that is somewhat of a standard in functional plugins and makes it easy to find which hook a function belongs to.